libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -193,20 +190,6 @@ auto get_re_configs(const std::string& _instance_name) -> irods::error
 					else {
 						audit_amqp_options = amqp_options_cfg->get<std::string>();
 					}
-
-					// look for a test mode setting.  if it doesn't exist just keep test_mode at false.
-					// if test_mode = true and log_path_prefix isn't set just leave the default
-					const auto test_mode_cfg = plugin_spec_cfg.find("test_mode");
-					if (test_mode_cfg != plugin_spec_cfg.end()) {
-						const auto& test_mode_str = test_mode_cfg->get_ref<const std::string&>();
-						test_mode = boost::iequals(test_mode_str, "true");
-						if (test_mode) {
-							const auto log_path_prefix_cfg = plugin_spec_cfg.find("log_path_prefix");
```

Comment:
Do we want to remove the `log_path_prefix` configuration option? Just making sure we've thought that through. All the log information available through this facility is probably available through an AMQP log somewhere, right?

We should remove this from the test suite as well.

Reply:
`log_path_prefix` was a `test_mode` thing. It serves no purpose without `test_mode`, which is removed in this PR (see #100).

Reply:
Understood. If we don't feel it is a useful concept even apart from `test_mode` then this is correct, yes.

Reply:
Now that I'm actually looking at the stuff in `packaging/`, I think I'll leave the removal of `test_mode` to someone else.